demo/scss/index.scss

```diff
@@ -93,7 +130,15 @@ input[type=text], select {
 }
 
 input[type=text]::placeholder {
-  color: var(--color-3);
+  color: var(--color-5);
+}
+
+select:required:invalid {
+  color: var(--color-5);
+}
+
+option[value=""][disabled] {
+  display: none;
 }
```

Comment:
we can remove this line and to something like this instead `<option value="" disabled selected  hidden>`

Reply:
Seems to work fine, I've submitted the change.